spot_examples/spot_examples/arm_wasd.py

```diff
@@ -0,0 +1,588 @@
+
+    def _arm_rotate_plus_ry(self) -> None:
+        self.arm_velocity_cmd_helper("arm_rotate_plus_ry", end_effector_y=ARM_VELOCITY_ANGULAR_NORMALIZED)
+        self.add_message("End effector: +Y rotation")
+
+    def _arm_rotate_minus_ry(self) -> None:
+        self.arm_velocity_cmd_helper("arm_rotate_minus_ry", end_effector_y=-ARM_VELOCITY_ANGULAR_NORMALIZED)
+        self.add_message("End effector: -Y rotation")
+
+    def _arm_rotate_plus_rx(self) -> None:
+        self.arm_velocity_cmd_helper("arm_rotate_plus_rx", end_effector_x=ARM_VELOCITY_ANGULAR_NORMALIZED)
+        self.add_message("End effector: +X rotation")
+
+    def _arm_rotate_minus_rx(self) -> None:
+        self.arm_velocity_cmd_helper("arm_rotate_minus_rx", end_effector_x=-ARM_VELOCITY_ANGULAR_NORMALIZED)
+        self.add_message("End effector: -X rotation")
+
+    def _arm_rotate_plus_rz(self) -> None:
+        self.arm_velocity_cmd_helper("arm_rotate_plus_rz", end_effector_z=ARM_VELOCITY_ANGULAR_NORMALIZED)
+        self.add_message("End effector: +Z rotation")
+
+    def _arm_rotate_minus_rz(self) -> None:
+        self.arm_velocity_cmd_helper("arm_rotate_minus_rz", end_effector_z=-ARM_VELOCITY_ANGULAR_NORMALIZED)
+        self.add_message("End effector: -Z rotation")
+
+    def arm_velocity_cmd_helper(
+        self,
+        desc: str = "",
+        r: float = 0.0,
+        theta: float = 0.0,
+        z: float = 0.0,
+        end_effector_x: float = 0.0,
+        end_effector_y: float = 0.0,
+        end_effector_z: float = 0.0,
+    ) -> None:
+        arm_cmd_request = arm_command_pb2.ArmVelocityCommand.Request()
+        arm_cmd_request.angular_velocity_of_hand_rt_odom_in_hand.x = end_effector_x
+        arm_cmd_request.angular_velocity_of_hand_rt_odom_in_hand.y = end_effector_y
+        arm_cmd_request.angular_velocity_of_hand_rt_odom_in_hand.z = end_effector_z
+
+        arm_cmd_request.maximum_acceleration.data = ARM_MAXIMUM_ACCELERATION
+        arm_cmd_request.command.command_choice = arm_cmd_request.command.COMMAND_CYLINDRICAL_VELOCITY_SET
+        arm_cmd_request.command.cylindrical_velocity.linear_velocity.r = r
+        arm_cmd_request.command.cylindrical_velocity.linear_velocity.theta = theta
+        arm_cmd_request.command.cylindrical_velocity.linear_velocity.z = z
+        arm_cmd_request.command.cylindrical_velocity.max_linear_velocity.data = ARM_MAX_LINEAR_VELOCITY
+
+        arm_cmd_request_ros = ArmVelocityCommandRequest()
+        convert(arm_cmd_request, arm_cmd_request_ros)
+        start_time = time.time()
+        while time.time() - start_time < ARM_VELOCITY_CMD_DURATION:
+            self.pub_arm_vel.publish(arm_cmd_request_ros)
+            time.sleep(0.01)
+        zero_arm_cmd_request = arm_command_pb2.ArmVelocityCommand.Request()
+        zero_arm_cmd_request_ros = ArmVelocityCommandRequest()
+        convert(zero_arm_cmd_request, zero_arm_cmd_request_ros)
+
+        self.pub_arm_vel.publish(zero_arm_cmd_request_ros)
```

Comment:
I think this part can be removed. It gets rid of terminal spew without affecting the performance. I would also reccomend setting the arm_is_unstowed bool to true here as this moves the arm

```suggestion
        self.is_arm_unstowed = True
```